src/main.sh

```diff
@@ -96,6 +97,10 @@ function main {
       installTerraform
       terraformApply "${*}"
       ;;
+    output)
+      installTerraform
+      terraformOutput
```

Comment:
This is to allow users to pass arguments to this subcommand.
```suggestion
      terraformOutput ${*}
```